src/v2/view-builder/views/IdentifierView.js

```diff
@@ -16,6 +17,33 @@ const Body = BaseForm.extend({
     return loc('oform.next', 'login');
   },
 
+  saveForm() {
+    // Ideally this can be added to a "preSaveForm" handler - but keeping this here for now.
+    if (!this.settings.get('features.deviceFingerprinting')) {
+      BaseForm.prototype.saveForm.apply(this, arguments);
+      return;
+    }
+
+    // Before the XHR is made for "identify", we'll generate this one-time use fingerprint via
+    // a hidden-iframe (similar to authn/v1 flows)
+    const fingerprintData = {
+      oktaDomainUrl: this.settings.get('baseUrl'),
+      element: this.$el,
+      stateHandle: this.options.appState.get('idx')?.rawIdxState?.stateHandle,
```

Comment:
does `this.options.appState.get('stateHandle')` or `this.options.currentViewState.stateHandle` give same result?
try to avoid `rawIdxState`